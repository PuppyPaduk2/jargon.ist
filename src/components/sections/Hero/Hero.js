// @flow

import React from 'react';

import { LinkButton } from '../../ui';

import hero from '../../../img/hero.svg';

import './Hero.scss';

const Hero = () => (
  <div className="c-hero row u-align-items-center">
    <div className="col col--lg-5">
      <h1 className="u-color-primary u-clear-gap-top u-gap-bottom-small">jargon.ist</h1>
      <p className="u-gap-bottom">
        Jargon.ist, bilgisayar bilimleri alanında kullanılan jargonların topluluk tarafından Türkçe
        olarak açıklanmasını sağlayan bir web uygulamasıdır.
      </p>

      <LinkButton to="/dizin" primary>
        Dizini Görüntüle
        <span className="u-pad-left-2xsmall">⟶</span>
      </LinkButton>
    </div>

    <div className="col col--lg-7 u-hidden@md-down u-text-right">
      <img src={hero} alt="jargon.ist" className="c-hero__img" />
    </div>
  </div>
);

export default Hero;
