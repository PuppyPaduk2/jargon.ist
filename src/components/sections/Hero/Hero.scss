@import '../../../styles/abstracts/index';

.c-hero {
  align-items: center;
  display: grid;
  grid-template-columns: 6fr 7fr;

  &__img {
    margin-left: auto;
  }
}
