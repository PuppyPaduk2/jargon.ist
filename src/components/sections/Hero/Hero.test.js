// @flow

import React from 'react';
import { shallow } from 'enzyme';

import Hero from './Hero';

it('renders correctly', () => {
  const wrapper = shallow(<Hero />);

  expect(wrapper).toMatchSnapshot();
});
