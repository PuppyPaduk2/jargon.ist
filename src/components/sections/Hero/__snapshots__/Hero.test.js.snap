// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="c-hero"
>
  <div>
    <h1
      className="u-color-primary"
    >
      Jargon.ist
    </h1>
    <p
      className="u-gap-bottom"
    >
      Jargon.ist, bilgisayar bilimleri alanında kullanılan bazı jargonların topluluk tarafından Türkçe olarak açıklanmasını sağlayan bir web uygulamasıdır.
    </p>
    <Button
      className=""
      primary={true}
      to="/dizin"
    >
      Dizini Görüntüle
      <span
        className="u-pad-left-2xsmall"
      >
        ⟶
      </span>
    </Button>
  </div>
  <img
    alt="jargon.ist"
    className="c-hero__img"
    src={
      Object {
        "process": [Function],
      }
    }
  />
</div>
`;
